examples/tests/NavigationSampleTests.elm

```diff
@@ -0,0 +1,35 @@
+module NavigationSampleTests exposing (all)
+
+import Expect
+import NavigationSample exposing (program)
+import Test exposing (..)
+import Test.Html.Event as Event
+import Test.Html.Query exposing (..)
+import Test.Html.Selector exposing (..)
+import TestContext exposing (..)
+
+
+all : Test
+all =
+    describe "NavigationSample"
+        [ test "renders navigation history" <|
+            \() ->
+                NavigationSample.program
+                    |> start
+                    |> simulate (find [ class "bears" ]) Event.click
+                    |> simulate (find [ class "cats" ]) Event.click
+                    |> expectView
+                    |> find [ class "history" ]
+                    |> Expect.all
+                        [ has [ text "#bears" ]
```

Comment:
yes, but it does, it is actually not the whole url in this case, just the pathname + the hash, like this:

![image](https://user-images.githubusercontent.com/792201/28348281-8dd5fe54-6c12-11e7-83cb-77aca7d934fc.png)

but, the `has [ text` matcher does not search for the exact whole string, it just tries to find this string piece anywhere. So for example, if I change this for some text that is not on the DOM, I can visualize the error and that's what I get:

![image](https://user-images.githubusercontent.com/792201/28348320-cd8ef9ec-6c12-11e7-9ad3-0ee837ae2afa.png)

You see, it is actually `/#cats` because the default `pathname` for the tests is `/`

I'll change this test to be more specific and include the `/`